api/src/OpenTelemetry/Internal/Logs/Types.hs

```diff
@@ -19,18 +20,92 @@ module OpenTelemetry.Internal.Logs.Types (
 
 import Control.Concurrent.Async
 import Data.Function (on)
+import Data.HashMap.Strict (HashMap)
```

Comment:
This replaces `H.HashMap`. Functions and things that aren't data types are still prefixed with `H.`.